keanu-project/build.gradle

```diff
@@ -40,6 +40,10 @@ dependencies {
 
 }
 
+compileJava {
+    options.compilerArgs += ['-parameters']
```

Comment:
Note: this makes the build 20k bigger but I think it's worth it if anyone is using IntelliJ to decompile and view the source.

Reply:
@CharlieCrisp found a way to not include this parameter argument!